### ISSUE TITLE
Volume Kernel not found flashing init

### ISSUE THREAD

**Issue by gericho** [4IceG/ZTE-MF286D#11]
```
eth0 PHY0 Down Speed :10 Half duplex
eth0 PHY1 up Speed :1000 Full duplex
eth0 PHY2 Down Speed :10 Half duplex
eth0 PHY3 Down Speed :10 Half duplex
eth0 PHY4 Down Speed :10 Half duplex
Using eth0 device
TFTP from server 192.0.0.1; our IP address is 192.132.35.204
Filename 'root_uImage'.
Load address: 0x84000000
Loading: #################################################################
         #################################################################
         #################################################################
         #################################################################
         #################################################################
         #################################################################
         #################################################################
         #################################################################
         #################################################################
         #################################################################
         #################################################################
         #################################################################
         #################################################################
         #################################################################
         #################################################################
         #################################################################
         #################################################################
         #################################################################
         #################################################################
         ############################
done
Bytes transferred = 18538264 (11adf18 hex)

NAND erase: device 0 offset 0x1000000, size 0x800000
Erasing at 0x17e0000 -- 100% complete.
OK

NAND write: device 0 offset 0x1000000, size 0x800000
 8388608 bytes written: OK

NAND erase: device 0 offset 0x1800000, size 0x1d00000
Erasing at 0x34e0000 -- 100% complete.
OK

NAND write: device 0 offset 0x1800000, size 0x1d00000
 30408704 bytes written: OK
zte_set_download_led(1) is called
blank download is done. send fin to PC...
eth0 PHY0 Down Speed :10 Half duplex
eth0 PHY1 up Speed :1000 Full duplex
eth0 PHY2 Down Speed :10 Half duplex
eth0 PHY3 Down Speed :10 Half duplex
eth0 PHY4 Down Speed :10 Half duplex
Volume kernel not found!
(IPQ40xx) #

```

**Comment by 4IceG** [comment]
Hi @gericho ,
Have you tested other initramfs-zImage.itb images from official OpenWrt releases?

You can install official OpenWrt image and update it to my image via sysupgrade (without saving the settings).
I don't know why the initramfs-zImage.itb image doesn't work, but I had such a router (to be repaired) and not every initramfs image started correctly.

**Comment by gericho** [comment]
Yes thank you! Partially "solved" by using the [23.05.2](https://downloads.openwrt.org/releases/23.05.2/targets/ipq40xx/generic/openwrt-23.05.2-ipq40xx-generic-zte_mf286d-initramfs-zImage.itb) and your sysupgrade. BTW I found the download is half the speed compared to the original Nordic B12 (~200Mbit vs ~110Mbits, upload speeds about the same), is there any chance to configure the QMI protocol (or other configs) to achieve the same Nordic B12 Speed? Tested by a Home Assistant automation that schedules a speed test every 15 minutes, so I am sure it is not caused by the provider. In modemband is see same aggregated bands B3 and B20 as the Nordic B12 (Wind-IT)

**Comment by 4IceG** [comment]
Unfortunately, there is a problem with performance degradation on OpenWrt.

Examples of topics with this problem:

Help with increase speed for 5G modem
[https://forum.openwrt.org/t/help-with-i … dem/168785](https://forum.openwrt.org/t/help-with-increase-speed-for-5g-modem/168785)
Zyxel NR7101 low performance
[https://forum.openwrt.org/t/zyxel-nr710 … nce/174991](https://forum.openwrt.org/t/zyxel-nr7101-low-performance/174991)
Low USB3 speed
https://forum.openwrt.org/t/low-usb3-speed/174044

Problem has recently been noticed because we haven't used such fast modems before. The search for a solution to this situation is ongoing. In my builds I try to add methods that are currently being tested.

**Comment by gericho** [comment]
All the posts you highlighted, refer to low-speed USB interfaces due to some CPU peaks caused by the Kmod driver ... if I understood correctly. So I installed HTOP to measure CPU peaks after uninstalling or disabling all the less important packages you included as aria2, adblock, smb and so on, the CPU now barely touches the 5% in idle, the only process that is a little "high" on CPU and intermittently is [wrtbwmon](https://openwrt.org/docs/guide-user/services/network_monitoring/wrtbwmon) and [ubus.sock](https://openwrt.org/docs/techref/ubus). Running the speedtest, the CPU2 reaches 100% whereas other cores stay around 40-60% **with no apparent process using it**. So yes, seems related also to CPU usage

What I did to increase the down speed by 40+Mbit
- removed all the "unnecessary" packages (to me)
- removed ipv6 DHCP on the wan (my ISP does not support it)
- network/interface/global network options/ ENABLED packet steering (does it use irqbalance?)
- disabled, reboot, enabled, reboot UPLOAD band aggregation in modemband

I am not sure what did the 60Mbit up-jump here, but it seems to fairly work, not 200Mbits as the Nordic B12 for now, but I'm reaching easily 150-170Mbits where before seemed to be capped at 90-110, kindly let me know. I'll study that patch on the [youtube video](https://openwrt.org/docs/guide-user/services/network_monitoring/wrtbwmon), not sure it applies to the mf286d too.

**Comment by 4IceG** [comment]
@gericho You can test the new build.

> * disabled, reboot, enabled, reboot UPLOAD band aggregation in modemband

If I understand correctly, you turned off band aggregation for upload, this should give a visible increase in speed.

**Comment by gericho** [comment]
In fact I disabled it but I don't have enough data collected to call it a statistic. So I re-enabled it. As mentioned in the previous post, I'm still not sure which change caused the speed increase. I'll investigate further. Thank you for the new release! I'll test it as soon as possible.

**Comment by gericho** [comment]
Unfortunately, the last version also has CPU caps somewhere (a speed test shows one CPU reaches 100% in htop). I also noticed the system/routing report non "advised" configuration in an orange banner on top, but I didn't dig deeper into that detail.
